clients/cobol-lsp-vscode-extension/src/services/copybook/downloader/CopybookDownloaderForE4E.ts

```diff
@@ -110,7 +112,12 @@ export class CopybookDownloaderForE4E {
     if (config) {
       return config;
     }
-    if (!this.e4e.isEndevorElement(uri)) return undefined;
+    if (
+      !this.e4e.isEndevorElement(uri) &&
+      SettingsService.getCopybookEndevorDependencySettings() !=
+        ENDEVOR_PROCESSOR
+    )
+      return undefined;
```

Comment:
This function (and other) should not be called in the first place by the download service. Just revert changes in this file, the change in the copybookdownloadservice.ts should suffice.